### PR TITLE
add harshit to contributors list

### PR DESCRIPTION
**Pull request by HarshitKumarOjha**

### CODE REVIEW

File: src/data/contributors/harshitkumarojha.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Harshit Ojha",
+    "skills" : ["html", "css", "bootstrap", "discord js", "React"],
+    "cause" : "Help making world become a better place to live :D",
+    "image" : "Avatar3",
+    "github" : "https://github.com/HarshitKumarOjha",
+    "discod" : "https://discord.gg/59HSBfT",
+    "quora" : "https://www.quora.com/profile/Harshit-Kumar-Ojha"
```
Comment (Abhishek-kumar09) [inline src/data/contributors/harshitkumarojha.json:8]: unfortunately, Discord and quora links can't be rendered on the web. Only github, link will be rendered.

Comment (HarshitKumarOjha) [reply src/data/contributors/harshitkumarojha.json]: removed the discord and quora links 🤗

Comment (HarshitKumarOjha) [reply src/data/contributors/harshitkumarojha.json]: sorry for disturbing but i just want to say that im a beginner and i did some mistakes...but now i fixed the code,
hope u'll merge the pr thank you :)

File: src/data/contributors/harshitkumarojha.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Harshit Ojha",
+    "skills" : ["html", "css", "bootstrap", "discord js", "React"],
+    "cause" : "Help making world become a better place to live :D",
+    "image" : "Avatar3",
+    "github" : "https://github.com/HarshitKumarOjha",
+    "twitter" : "@Jet_Hub"
```
Comment (Abhishek-kumar09) [inline src/data/contributors/harshitkumarojha.json:7]: You have to provide the full url to your twitter profile.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Comment by Abhishek-kumar09** [comment]
Thanks for your PR, hoping you have get your PR marked as a valid contribution in hacktoberfest.

**Pull request merged**